spec.emu

```diff
@@ -331,6 +359,45 @@ contributors: Google, Ecma International
         The value of the [[RelevantExtensionKeys]] internal slot is &laquo;  &raquo;.
       </p>
 
+      <p>
+        The value of the [[LocaleData]] internal slot is implementation defined within the constraints described in <emu-xref href="#sec-internal-slots"></emu-xref> and the following additional constraints:
+      </p>
+
+      <ul>
+        <li>
+        [[LocaleData]].[[&lt;_locale_&gt;]] must have a types field for all locale values _locale_.
+        The value of this field must be a Record, which must have fields with the names of the six display name types:
+        `"language"`, `"region"`, `"script"`, `"currency"`, `"dateTime"`, and `"dateTimeField"`.
+        </li>
+        <li>The field `"language"` noted above must be Records which must have fields with the names of the three display name styles:
+        `"narrow"`, `"short"`, and `"long"`. The value of these fields in turn must be a Record, which must have fields with keys
+        corresponding to language codes according to. The value of these fields
+        must be string values.
+        </li>
+        <li>The field `"region"` noted above must be Records which must have fields with the names of the three display name styles:
+        `"narrow"`, `"short"`, and `"long"`. The value of these fields in turn must be a Record, which must have fields with keys
```

Comment:
Is there a way you can avoid duplicating the three display name styles in each list item?  Something along the lines of,

> 1. ...
> 1. The value of this field must be a Record, which must have fields with the names of the six display name types: `"language"`, `"region"`, `"script"`, `"currency"`, `"dateTime"`, and `"dateTimeField"`.
> 1. The value of each of those fields in turn must be a Record that has fields corresponding to the three display name styles: `"narrow"`, `"short"`, and `"long"`.
> 1. The three display name styles fields under display name type `"language"` should contain Records, with keys corresponding to language codes according to ___. The value of these fields must be string values.
> 1. ...